### ISSUE TITLE
Complete first round of HTML validation

### ISSUE THREAD

**Issue by mamf92** [mamf92/semester-project-1#20]
Use deployed website and go through each page with W3 HTML validation.

- [ ] Fix errors
- [ ] Fix warnings if necessary

**Comment by mamf92** [comment]
Fixed all errors. For accessibility H1 and H2 headings should be considered added to main and sections respectively.
